flake.nix

```diff
@@ -116,6 +116,8 @@
           wasm-pack # for repl_wasm
           jq # used in several bash scripts
           cargo-nextest # used to give more info for segfaults for gen tests
+          nodejs_18 # for npx in `build-dev-local`
+          zls # zig language server
```

Comment:
@Anton-4 any concerns about including these?

Reply:
Yes, I think we should ditch npx and use [simple-http-server](https://crates.io/crates/simple-http-server) like before. I'll check on zulip if that's ok.

Including zls seems pretty reasonable.

Thanks for alerting me Folkert :)

Reply:
> I think we should ditch npx and use [simple-http-server](https://crates.io/crates/simple-http-server) like before. I'll check on zulip if that's ok.

Richard said it's ok, he was having some performance problem with the feedback loop working on wasm stuff with the simple-http-server. But he's fine with going back to simple-http-server.

Reply:
We should do that in separate PR though

Reply:
sounds good, ill remove node and keep zls